### ISSUE TITLE
PDF breaks badly if body size is defined with CSS.

### ISSUE THREAD

**Issue by rredesigns** [dompdf/dompdf#3061]
Hi there.

I was tweaking my HTML to design my document and then I wanted to create a PDF, but the size of the content was terribly off. Here's what I was using:

![image](https://user-images.githubusercontent.com/10718720/201550196-110ad0cb-c08c-4753-aea4-fd792e6edcc1.png)

It served well to position everything visually, but the document always looked like this:

![image](https://user-images.githubusercontent.com/10718720/201550272-18ace97e-f9af-4192-b71b-eb2413cf85a5.png)

I spent the whole day going crazy about this and the issue is just that piece of CSS. So is there any way to either tweak the code to ignore that or at least add this to the docs or the known-issues list? Or is this actually expected behaviour?

**Comment by bsweeney** [comment]
You don't specify the PDF page size so I'm going to assume you're using the default of "letter" or 8.5in x 11in (215.9mm x 279mm). With 1in margins you have a usable space of 190.5mm x 253.6. Obviously wider than the page.

Also, you have some padding there which will also expand the container width since Dompdf does not yet support `box-sizing: border-box` (#669).

**Comment by rredesigns** [comment]
I was going for an A4 page, so I made my HTML an A4 page sized design. I figured the library would take that and put it in a PDF file.

Then I read a comment on Stack Overflow from somebody saying "if you don't want the default margins..." and then it hit me, but it took me like half a day to figure it out.  

Maybe I'm just dumb, since I didn't see anyone else having the same problem, but I don't know, maybe a "mind the default margins" in the docs would have been very useful, for me at least.

Cheers. :)
